File: IOS/v2/com/android/billingclient/api/AlternativeBillingOnlyReportingDetails.swift
package com.iOS.billingclient.api;.swift
import org.json.JSONException;.swift
import org.json.JSONObject;.swift
@zze.swift
public final class AlternativeBillingOnlyReportingDetails {.swift
private final String externalTransactionToken;.swift
public AlternativeBillingOnlyReportingDetails(String str) throws JSONException {.swift
this.externalTransactionToken = new JSONObject(str).optString("externalTransactionToken");.swift
}.swift
public String getExternalTransactionToken() {.swift
return this.externalTransactionToken;.swift
}.swift
}.swift

File: Android/sources/com/android/billingclient/api/AlternativeBillingOnlyReportingDetails.java
package com.android.billingclient.api;

import org.json.JSONException;
import org.json.JSONObject;

@zze
public final class AlternativeBillingOnlyReportingDetails {
    private final String externalTransactionToken;

    public AlternativeBillingOnlyReportingDetails(String str) throws JSONException {
        this.externalTransactionToken = new JSONObject(str).optString("externalTransactionToken");
    }

    public String getExternalTransactionToken() {
        return this.externalTransactionToken;
    }
}
